requirements/extras/dynamodb.txt

```diff
@@ -1 +0,0 @@
-boto3>=1.4.6
```

Comment:
Please don't do this. This affects the requirements of those extras when installing with pip.

Reply:
this PR was working well until I rebased on master eb375951c72c7e47604a473bcb87b36e8cdda1a0 at that time - the changes introduced by  @auvipy motivated my cleaning on the requriements...

Why? I got this error on the build:
```
Step 23/26 : RUN pyenv exec python3.7 -m pip install   -r requirements/default.txt   -r requirements/test.txt   -r requirements/test-ci-default.txt   -r requirements/docs.txt   -r requirements/test-integration.txt   -r requirements/pkgutils.txt &&   pyenv exec python3.6 -m pip install   -r requirements/default.txt   -r requirements/test.txt   -r requirements/test-ci-default.txt   -r requirements/docs.txt   -r requirements/test-integration.txt   -r requirements/pkgutils.txt &&   pyenv exec python3.5 -m pip install   -r requirements/default.txt   -r requirements/test.txt   -r requirements/test-ci-default.txt   -r requirements/docs.txt   -r requirements/test-integration.txt   -r requirements/pkgutils.txt &&   pyenv exec python3.4 -m pip install   -r requirements/default.txt   -r requirements/test.txt   -r requirements/test-ci-default.txt   -r requirements/docs.txt   -r requirements/test-integration.txt   -r requirements/pkgutils.txt &&   pyenv exec python2.7 -m pip install   -r requirements/default.txt   -r requirements/test.txt   -r requirements/test-ci-default.txt   -r requirements/docs.txt   -r requirements/test-integration.txt   -r requirements/pkgutils.txt
 ---> Running in b46583a53ada
Ignoring couchbase-cffi: markers 'platform_python_implementation == "PyPy"' don't match your environment
Double requirement given: boto3>=1.4.6 (from -r requirements/extras/dynamodb.txt (line 1)) (already in boto3>=1.9.125 (from -r requirements/test.txt (line 3)), name='boto3')
```


so to fix it I just removed those duplicated dependencies.

And I don't know if I broke anything but tox is not complaining when building the environments:
```
tox -e 3.6-integration-dynamodb
3.6-integration-dynamodb create: /home/developer/celery/.tox/3.6-integration-dynamodb
3.6-integration-dynamodb installdeps: -r/home/developer/celery/requirements/default.txt, -r/home/developer/celery/requirements/test.txt, -r/home/developer/celery/requirements/docs.txt, -r/home/developer/celery/requirements/pkgutils.txt, -r/home/developer/celery/requirements/test-ci-default.txt, -r/home/developer/celery/requirements/test-integration.txt
WARNING: Discarding $PYTHONPATH from environment, to override specify PYTHONPATH in 'passenv' in your configuration.
3.6-integration-dynamodb develop-inst: /home/developer/celery
WARNING: Discarding $PYTHONPATH from environment, to override specify PYTHONPATH in 'passenv' in your configuration.
3.6-integration-dynamodb installed: alabaster==0.7.12,amqp==2.4.2,asn1crypto==0.24.0,atomicwrites==1.3.0,attrs==19.1.0,aws-xray-sdk==0.95,azure-common==1.1.5,azure-nspkg==3.0.2,azure-storage==0.36.0,azure-storage-common==1.1.0,azure-storage-nspkg==3.1.0,Babel==2.6.0,basho-erlastic==2.1.1,billiard==3.6.0.0,boto==2.49.0,boto3==1.9.126,botocore==1.12.126,bumpversion==0.5.3,case==1.5.3,cassandra-driver==3.17.0,-e git://github.com/celery/celery.git@dc8c387dc4f7b9751e6e97db33a70f28a9d4b2ae#egg=celery,certifi==2019.3.9,cffi==1.12.2,chardet==3.0.4,codecov==2.0.15,couchbase==2.5.4,coverage==4.5.3,cryptography==2.6.1,DateTime==4.3,dnspython==1.16.0,docker==3.7.2,docker-pycreds==0.4.0,docutils==0.14,ecdsa==0.13,elasticsearch==6.3.1,entrypoints==0.3,ephem==3.7.6.0,eventlet==0.24.1,filelock==3.0.10,flake8==3.7.7,future==0.17.1,gevent==1.4.0,greenlet==0.4.15,idna==2.8,imagesize==1.1.0,Jinja2==2.10,jmespath==0.9.4,jsondiff==1.1.1,jsonpickle==1.1,kombu==4.5.0,linecache2==1.0.0,MarkupSafe==1.1.1,mccabe==0.6.1,mock==2.0.0,monotonic==1.5,more-itertools==7.0.0,moto==1.3.7,msgpack==0.6.1,nose==1.3.7,packaging==19.0,pbr==5.1.3,pluggy==0.9.0,py==1.8.0,pyaml==18.11.0,pyArango==1.3.2,pycodestyle==2.5.0,pycouchdb==1.14,pycparser==2.19,pycryptodome==3.8.0,pycurl==7.43.0.2,pydocstyle==1.1.1,pydocumentdb==2.3.2,pyflakes==2.1.1,Pygments==2.3.1,pylibmc==1.6.0,pymongo==3.7.2,pyparsing==2.3.1,pytest==4.4.0,pytest-cov==2.6.1,pytest-sugar==0.9.2,pytest-travis-fold==1.3.0,python-consul==1.1.0,python-dateutil==2.8.0,python-jose==2.0.2,python-memcached==1.59,python3-protobuf==2.5.0,pytz==2018.9,PyYAML==5.1,redis==3.2.1,requests==2.21.0,responses==0.10.6,riak==2.7.0,s3transfer==0.2.0,simplejson==3.16.0,six==1.12.0,snowballstemmer==1.2.1,softlayer-messaging==1.0.3,Sphinx==1.8.5,sphinx-celery==1.4.6,sphinx-testing==0.7.2,sphinx2rst==1.1.0,sphinxcontrib-websupport==1.1.0,SQLAlchemy==1.3.1,tblib==1.3.2,termcolor==1.1.0,toml==0.10.0,tox==3.8.4,traceback2==1.4.0,typing==3.6.6,unittest2==1.1.0,urllib3==1.24.1,vine==5.0.0a1,virtualenv==16.4.3,websocket-client==0.56.0,Werkzeug==0.15.1,wrapt==1.11.1,xmltodict==0.12.0,zope.interface==4.6.0
3.6-integration-dynamodb run-test-pre: PYTHONHASHSEED='3274017391'
3.6-integration-dynamodb run-test: commands[0] | py.test -xsv t/integration
Test session starts (platform: linux, Python 3.6.7, pytest 4.4.0, pytest-sugar 0.9.2)
cachedir: .tox/3.6-integration-dynamodb/.pytest_cache
rootdir: /home/developer/celery, inifile: setup.cfg
plugins: travis-fold-1.3.0, sugar-0.9.2, cov-2.6.1, case-1.5.3, celery-4.3.0
collecting ... 
```

(but I'm not sure since it seems it hangs at `collecting ...`)

in any case either the requirements files are wrong or the Dockerfile is wrong... any suggestion?

Thanks @thedrow !

Reply:
The requirements files are wrong.
We can fix that in another PR.

If the integration test suite hangs it probably means it can't connect to the broker.

Reply:
@thedrow what do you suggest then? should I just drop this commit: https://github.com/celery/celery/pull/5422/commits/6d76b448b646723ba42b3ec99ddd8c2f648dce16  and we fix the boto3 issue in another PR, right?

Reply:
Yes.
That's fine by me.

Reply:
done